setup.py

```diff
@@ -28,7 +28,7 @@
 
 setup(
 name='fiware-glancesync',
-packages=['fiware-glancesync'],  # this must be the same as the name above
+packages=['glancesync'],  # this must be the same as the name above
```

Comment:
I prefer to specify which one no and not which yes.

I mean:

  packages=find_packages(exclude=['*tests*']),